master/static/migrations/20210429141534_add-trial-metadata.up.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE public.trials
+    ADD COLUMN run_start timestamp without time zone NULL,
+    ADD COLUMN metadata jsonb NOT NULL DEFAULT '{}'::jsonb;
+
+AlTER TABLE public.steps
+    ADD COLUMN total_inputs integer NOT NULL DEFAULT 0;
```

Comment:
total_inputs is a bit ambiguous. It doesn't immediately deliver it is total_batches or total_inputs of the model

Reply:
yeah I agree. total_records?

Reply:
yeah. that makes sense in using the units to let us know it's the number of examples. but I thought we used batches?

Reply:
i think sprinkle proposes to have both available, but this one is optional